src/test/java/namor/template/calculator/CalculatorFactoryTest.java

```diff
@@ -0,0 +1,40 @@
+package namor.template.calculator;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+public class CalculatorFactoryTest {
+
+    private CalculatorFactory calculatorFactory;
+
+    @Test
+    public void whenInputRpnExpressionReturnRPNCalculator() {
+        final String input = "1 1 +";
+        final Calculator actual;
+        calculatorFactory = new CalculatorFactory();
```

Comment:
Get familiar with `@Before` annotation and make set up method where you'll create the tested instance - `new CalculatorFactory()`